<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="menu_tw">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1011</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tables_tab">
     <attribute name="title">
      <string>Tables</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
       <item>
        <widget class="QLabel" name="table_lbl">
         <property name="text">
          <string>Choix de la table :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="table_cb"/>
       </item>
       <item>
        <widget class="QPushButton" name="submit_table_cb_btn">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="table_tv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>971</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stats_tab">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>971</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="score_tab">
       <attribute name="title">
        <string>Score Equipe</string>
       </attribute>
       <widget class="QTableView" name="equipe_tv">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>931</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="stade_tab">
       <attribute name="title">
        <string>Meilleur Stade</string>
       </attribute>
       <widget class="QTableView" name="stade_tv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>531</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="stat_generator_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Générer les Statistiques</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admin_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Administration</string>
     </attribute>
     <widget class="QTabWidget" name="admin_site_tab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>971</width>
        <height>671</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Calendrier</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>831</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="2">
          <widget class="QComboBox" name="equipeB_cb">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="arbitre_cb_lbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Choix de l'Arbitre: </string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="create_calendrier_btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Créer</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="eqB_cb_lbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Choix de l'Equipe B: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="equipeA_cb">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="create_form_calendrier_btn">
           <property name="text">
            <string>Prévoir un nouveau match</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="stade_cb_lbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Choix du stade:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="arbitre_cb">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="stade_cb">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="eqA_cb_lbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Choix de l'Equipe A: </string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QDateTimeEdit" name="calendrier_date_de">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Match</string>
       </attribute>
       <widget class="QTableView" name="match_admin_tv">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>911</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="match_edit_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>570</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbreSpectateurs_le">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>570</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbreButsA_le">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>570</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbreButsB_le">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>570</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nbr buts ClubA</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>550</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nbr buts ClubB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>550</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre de Spectateurs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refresh_match_btn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraîchir</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="params_tab">
     <attribute name="title">
      <string>Paramètres</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>19</y>
        <width>531</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,2,1">
       <item>
        <widget class="QLabel" name="username_lbl">
         <property name="text">
          <string>Nom D'utilisateur : {USERNAME}</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect_btn">
         <property name="text">
          <string>Se Déconnecter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
